client/src/main.ts

```diff
@@ -14,9 +15,27 @@ import router from './router'
 
 const app = createApp(App)
 
+const csfCustomTheme: ThemeDefinition = {
+  dark: false,
+  colors: {
+    red: 'rgb(237, 28, 36)',
+    black: 'rgb(0, 0, 0)',
+    white: 'rgb(255, 255, 255)',
+    green: 'rgb(0, 157, 79)',
+    blue: 'rgb(52, 94, 158)',
+    tint: 'rgb(249, 241, 227)'
+  }
```

Comment:
I would give them better names
Like red = primary-red
black = primary-black
blue = secondary-blue
green = secondary-green
etc :)

It will be good to merge after